src/mame/drivers/raiden_ms.cpp

```diff
@@ -652,7 +652,7 @@ WRITE_LINE_MEMBER(raiden_ms_state::vblank_irq)
 
 void raiden_ms_state::raidenm(machine_config &config)
 {
-	/* basic machine hardware */
+	// Basic machine hardware
```

Comment:
tbh I'm not a fan of these "basic machine / video / sound hardware" notes.
It's just noise that roots back when MAME source wasn't readable as it is now.

Reply:
You mean that it's better to fully remove them?

Reply:
It's subjective, leave it up to the driver author.

Reply:
I'd even go as far as saying "Basic" vs "basic" (eg. single line comments starting with a capital letter or not) is subjective as well.

Reply:
> I'd even go as far as saying "Basic" vs "basic" (eg. single line comments starting with a capital letter or not) is subjective as well.

I tend to not capitalise the first letter of a comment when it’s a grammatical fragment (i.e. not a complete sentence).  I don’t appreciate having them all made uppercase arbitrarily.

Reply:
imho it's really cargo cult that is unneeded. Some caveats:
1. What even "basic machine hardware" means, does the attached CPU can run a BASIC flavour? 😃
2. It's not really standardized in source (there are `// Basic`, `/* basic */` and variants of them;
3. If I really have to search in source, I'd rather search for a `machine_config` instead;
4. In worst case scenario where dev has to split video / sound fragments into functions he totally can nowadays. And it sounds dumb to do something among the lines of:
```c++
// video hardware
video_config(config);
```
Instead of just using the name of the function to identify it out.